README.md

```diff
@@ -94,6 +94,15 @@ installed.
 
         bundle exec guard
 
+-   [`rspec-snapshot](https://github.com/levinmr/rspec-snapshot)` is
+    used for some of the tests.  To update the snapshots, do:
+
+        UPDATE_SNAPSHOTS=true bundle exec rspec
+
+    or just for a subset of the tests:
+
+        UPDATE_SNAPSHOTS=true bundle exec rspec spec/foo/bar
```

Comment:
Nice, I knew there was a feature like that somewhere, I've just been deleting the snapshots that are obviously outdated. :)